modin/core/execution/dask/implementations/pandas_on_dask/partitioning/virtual_partition.py

```diff
@@ -93,16 +96,7 @@ def deploy_axis_func(
             *partitions,
             pure=False,
         )
-
-        lengths = kwargs.get("_lengths", None)
-        result_num_splits = len(lengths) if lengths else num_splits
-
-        # We have to do this to split it back up. It is already split, but we need to
-        # get futures for each.
-        return [
-            client.submit(lambda l: l[i], axis_result, pure=False)
-            for i in range(result_num_splits * 4)
-        ]
+        return axis_result
```

Comment:
```suggestion
        return result
```
We can probably return just DaskWrapper.deploy(...).